### PR TITLE
Ibm docs

### PR DESCRIPTION
**Pull request by mitra**
These changes on the branch is visible on the alternate doc server at

http://ec2-34-222-41-154.us-west-2.compute.amazonaws.com/

## done
54) Michael appreciated documentation work
    - Thank you no action item here.
55) observed - no register plugin screen present in doc
    - added more highlight to the login/ logout aspect.
    - done
57) doc - appliance took around 10 min to get IPs after power on
    - done 
59) doc - add cloud info is not in deployment planning template
    - done
60) doc - create a user(piouser) in vcd for service
    - done
61) doc - State cleary about deployment mode, don't leave it to the user (standard or cluster)
   - can't do this have to give the choice where it exist. If we want to remove it
     we might as well remove it from the product.
62) doc - In network planning, we have mentioned the only network while we created 3 networks.... do we need to document that as well?
    - Documentation in multi-network planning has all three.
64) doc - Mention ETA for cloud deployment
    - after submit tell user that it will be stuck at 1% for about 25 minutes.
    - next gets stuck at 50% for another 25 minutes/

## pending
58) doc - VPN ETA for operation
    - Need to give more detailed instructions and steps;
63) doc - Inconsistency in on-prem and on-cloud keyword (on-premise, few small, few camel cases)
 65) back links to network planning page.

### CODE REVIEW

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md [hunk 1]
```diff
@@ -104,3 +104,47 @@ _Table 1: Resource Requirements_
...
+**Identify the oprganization ID**
+
+* Loging to the tenant portal
+* The Organization ID can be identified from the location in  _Figure 1_
+
+_Figure 1: Screenshot show how to identify Organization ID_
+![Organization Id](images/Organization-id.png?classes=content-img "Screenshot showing how to find organization id") 
+
+
+**Create new user with admin priveledges**
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: spell check - privileges

Also, we should call our Organization Admin privileges

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md [hunk 2]
```diff
@@ -104,3 +104,47 @@ _Table 1: Resource Requirements_
...
+
+We do not recommend using the org administrator credentials directly
+while deploying HDM. Instead we recommend that the steps to create a new user with
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: recommend **that** the steps - need to remove that.

Can we rephrase it as - Instead we recommend the creation of a new organization admin for HDM integration. We will refer to this new admin in the rest of the document.

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md [hunk 3]
```diff
@@ -104,3 +104,47 @@ _Table 1: Resource Requirements_
...
+
+
+**Create new user with admin priveledges**
+
+We do not recommend using the org administrator credentials directly
+while deploying HDM. Instead we recommend that the steps to create a new user with 
+administrator privileges and use it in rest of the document.
+
+1. Create the new admin user using procedure [here to create new user.](https://docs.vmware.com/en/VMware-Cloud-Director/10.1/VMware-Cloud-Director-Tenant-Portal-Guide/GUID-1CACBB2E-FE35-4662-A08D-D2BCB174A43C.html) 
+1. Make sure you give it the **Organization Administrator* role see _Figure 2._
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: Can notice an extra or missing *

File: pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md [hunk 4]
```diff
@@ -152,7 +152,9 @@ _Figure 10: Viewing the On-Premises vCenter on the HDM Appliance_
 
 
 1. Select  **Register** to register the HDM plug-in to on-premises vCenter.
-2. Logout, then login twice from the vCenter for the plug-in installation to complete. Successful installation should add an HDM icon to the vCenter home page (figure 11). Sometimes, multiple login/logout cycles may be required for this step to complete.
+2. **Logout, then login twice** from the vCenter for the plug-in installation to complete. Successful installation should add an HDM icon to the vCenter home page (figure 11). Sometimes, multiple login/logout cycles may be required for this step to complete.
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md]: Logout and login twice

File: pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md [hunk 5]
```diff
@@ -391,6 +393,13 @@ _Figure 27: Network Mapping_
 
 10. To submit the request and initiate the cloud deployment, select _FINISH_. This will create deployment tasks for the cloud components, which can be monitored in vCenter Tasks. The progress can also be seen in the HDM Clouds view (figure 29).
 
+>>> **Attention**: 
+>>> After you submit the request the cloud deployment can take from 45 minutes to 
+>>> over an hour depending on storage and network performance. Please note
+>>> that the progress bar desplayed during this process is not linear and 
+>>> at 1% it takes about 25 minutes and at 50% another
```
Comment (sumitkapoor) [inline pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md]: At 1% it takes 40-50 Mins for HDM on-cloud deployment task, as we upload the required packages to Organization VDC. Under Organization VDC upload ISO stays at 50% for about 25 mins and then upload OVF stays at 1% for about 20 mins.

**Review by sumitkapoor — CHANGES_REQUESTED** [review]

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md (hunk 1 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: fixed

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md (hunk 2 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: done

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md (hunk 3 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: fixed

File: pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md (hunk 4 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md]: fixed

File: pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md (hunk 5 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md]: Rather elaborate and complicated to spell out the times and percentages, but have made the change never the less.

fixed.

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md [hunk 6]
```diff
@@ -104,3 +104,46 @@ _Table 1: Resource Requirements_
...
+![Organization Id](images/Organization-id.png?classes=content-img "Screenshot showing how to find organization id") 
+
+
+**Create new user with Organization Administrator priveleges**
+
+We recommend the creation of a new organization administrator for HDM integration.
+We refer to this administrator in the rest of the documentation.
+
+1. Create the new admin user using procedure [here to create new user.](https://docs.vmware.com/en/VMware-Cloud-Director/10.1/VMware-Cloud-Director-Tenant-Portal-Guide/GUID-1CACBB2E-FE35-4662-A08D-D2BCB174A43C.html) 
+1. Make sure you give it the **Organization Administrator** role see _Figure 2._
```
Comment (ayyajpio) [inline pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md:132]: Point numbering (1) repeated.

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md [hunk 7]
```diff
@@ -104,3 +104,46 @@ _Table 1: Resource Requirements_
...
+_Figure 1: Screenshot show how to identify Organization ID_
+![Organization Id](images/Organization-id.png?classes=content-img "Screenshot showing how to find organization id") 
+
+
+**Create new user with Organization Administrator priveleges**
+
+We recommend the creation of a new organization administrator for HDM integration.
+We refer to this administrator in the rest of the documentation.
+
+1. Create the new admin user using procedure [here to create new user.](https://docs.vmware.com/en/VMware-Cloud-Director/10.1/VMware-Cloud-Director-Tenant-Portal-Guide/GUID-1CACBB2E-FE35-4662-A08D-D2BCB174A43C.html)
```
Comment (ayyajpio) [inline pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: inconsistent use of the word 'admin' and 'administrator' in many places.

**Review by sumitkapoor — APPROVED** [review]

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md (hunk 6 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md:132]: Ayyaj in MD a numbered list is written like this

```
1. asdfsaf
1. asdfasf
```
MD renders them numerically, see the rendering in 

http://ec2-34-222-41-154.us-west-2.compute.amazonaws.com/hdm%20documentation/vcd/deployment%20planning

Screen Shot 2020-11-06 at 3 26 44 PM

File: pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md (hunk 7 above)
Comment (mitra) [reply pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md]: fixed.

**Review by ayyajpio — APPROVED** [review]
Looks good to me.

**Pull request merged**